src/main/java/de/unistuttgart/overworldbackend/controller/NPCInputController.java

```diff
@@ -0,0 +1,29 @@
+package de.unistuttgart.overworldbackend.controller;
+
+import de.unistuttgart.overworldbackend.data.PlayerNPCStatisticDTO;
+import de.unistuttgart.overworldbackend.data.PlayerNPCStatisticData;
+import de.unistuttgart.overworldbackend.service.PlayerNPCStatisticService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import javax.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@Tag(name = "Submit game data", description = "Submit a NPC statistic")
+@RestController
+@Slf4j
+@RequestMapping("/internal")
```

Comment:
Perhaps under 
```suggestion
@RequestMapping("/statistics/npcs")
```
instead?